src/goto-symex/symex_target_equation.h

```diff
@@ -168,6 +170,33 @@ class symex_target_equationt:public symex_targett
 
   exprt make_expression() const;
 
+  /// Single SSA step in the equation. Its \c type is defined as
+  /// goto_trace_stept::typet. Every SSA step has a \c source to identify its
+  /// origin in the input GOTO program and a \c guard expression which holds
+  /// the path condition required to reach this step: they limit the scope of
+  /// this step.
+  ///
+  /// SSA steps that represent assignments and declarations also store the
+  /// left- and right-hand sides of the assignment. The left-hand side
+  /// \c ssa_lhs is required to be of type \c ssa_exprt: in SSA form, variables
+  /// are only assigned to once. To achieve that, we annotate the original name
```

Comment:
:pick: `are only assigned once` (without the `to`)

Reply:
:bulb: Might be useful to link to `\ref static-single-assignment` here (SSA documentation in the `goto_symex` folder readme)

Reply:
Done.